src/SDL2/SDL2_GameWindow.cs

```diff
@@ -268,12 +271,18 @@ int clientHeight
 				// If exiting fullscreen, just center the window on the desktop.
 				SDL.SDL_SetWindowPosition(
 					INTERNAL_sdlWindow,
-					SDL.SDL_WINDOWPOS_CENTERED,
-					SDL.SDL_WINDOWPOS_CENTERED
+					INTERNAL_lastWindowPosition.X,
+					INTERNAL_lastWindowPosition.Y
 				);
 			}
 			else if (!INTERNAL_wantsFullscreen)
 			{
+				// Store the last window position
+				INTERNAL_lastWindowPosition = new Point(
```

Comment:
If you want to keep this particularly lean, you can assign the original Point's X/Y values instead of using `new` or simply use two ints, avoiding the Point struct entirely. I'd be okay with either one.